CIP-_/README.md

```diff
@@ -0,0 +1,209 @@
+    -   Next we add all of our selected Utxos to the global selected Utxos list and remove the newly selected Utxos from the candidate Utxos list!
+
+-   <b>Step 3: Change Selection Function</b>
+
+The Change Selection function creates change outputs that go back to the payment address that contains all of the extra assets and ada that is not used in the outputs of this transaction. This function will have the following parameters:
+
+    -   Our intial coin selection object
+    -   A Balance object containing the aggregated asset
+    -   A string change address that the change assets will go back to
+    -   An integer representing the fee buffer
+
+Example Definition (C#):
+
+    public void CalculateChange(CoinSelection coinSelection, Balance outputBalance, string changeAddress, ulong feeBuffer = 0) { }
+
+The idea behind this function is to create output Utxos with change that limit the size of future transactions by splitting large Utxos or Utxos with a lot of assets. For this we introduce 2 constant parameters that can be adjusted to fit your algorithm needs, the maxChangeOuputs which we set to 4, and the idealMaxAssetsPerOutput which we set to 30.
+
+The maxChangeOutputs parameter indicates that we have a maximum of 4 change outputs unless we require additional outputs due to those ideal 4 reaching their size limits. The idealMaxAssetsPerOutput is the ideal maximum number of assets in an output Utxo before we create a new output. This value is set to 30 meaning we try to output at max 30 assets in a change output. Having more then 30 is fine but the algorithm attempts to balance Utxos such that it does not exceed 30 if we can avoid it.
+
+-   <b>Step 3a)</b>
+
+    -   We first want to create a list of assets, sorted by policy ids. The reason for this is that we want to keep assets of the same policy ids in the same output Utxos if possible, even if the amount of assets go above 30 per output. This is because assets are grouped by policy ids in outputs and each output would need to duplicate the policy id data which is uneccesary extra bytes of overhead.
+    -   We are then going to loop through each asset and add it to the most recently created output. If the assets in that output go above the ideal assets per output we create a new output unless the asset is part of an existing policy id. If a change output is over 2kb we create a new output automatically. The max output size on Cardano is 5kb but we are using 2kb to have more evenly split Utxos in users wallet for smaller future transactions.
+
+-   <b>Step 3b)</b>
+
+    -   After looping through each asset we have a list of change outputs. We want to calculate the ada minUtxos required for all of those change outputs.
+    -   Using this ada minUtxo number along with the amount of ada in the balance we attempt to evenly distribute ada accross the existing change outputs, with the remainder going to the final change output.
+
+## Path to Active
```

Comment:
@NicholasMaselli can you reformat this section clearly with checkboxes on each item, like you see for example [in CIP-0095](https://github.com/cardano-foundation/CIPs/tree/master/CIP-0095#path-to-active), with the common sections grouped into headings which each have their own checkbox to represent total completion of all items?

Editors, reviewers & implementors should be able to scan down the list & quickly see what's been done & pending.  Any confirmation or qualification of how external implementations relate to the algorithm you are proposing should be documented in the _Rationale_ section instead.

In fact, your CIP doesn't have a `## Rationale` section, which is mandatory.  So I would recommend moving the SaturnNFT and Levvy material into _Rationale_ (immediately before) & addressing how the prior art relates to the algorithm that you put into your own reference implementation & want to bring to Cardano at large.

See `Rationale: how does this CIP achieve its goals?` ([CIP-0001 > Structure](https://github.com/cardano-foundation/CIPs/blob/master/CIP-0001/README.md#structure))